crosslink/README.md

```diff
@@ -128,3 +130,9 @@ Can be disabled when overwriting.
 
 **Quick Tip: Make sure your `go.mod` files are tracked and committed in a VCS
 before running crosslink.**
+
+### work
```

Comment:
The README should also highlight the available flags and their default values.

Reply:
I think only the `--go` flag misses description. Am I correct? 

PS. I think it might be good also to describe the `help` command, but I would prefer to do it in a separate PR.

Reply:
Yup, I think that's it. 

Yes totally agreed. I believe `help` is built in with cobra right? I think this is just a matter of doc philosophy. Do we want users to have to install our application before they are able to see configuration options?! 

Off topic: I wonder if cobra could generate docs for us on flags and what not. That would be nice.

Reply:
> I believe help is built in with cobra right? 

Correct.

> Off topic: I wonder if cobra could generate docs for us on flags and what not. That would be nice.

Yes it is possible: https://github.com/spf13/cobra/blob/main/doc/md_docs.md

We could probably even start by improving the help output and dump it in `README.md`. This is how README for [semconvgen](https://github.com/open-telemetry/opentelemetry-go-build-tools/tree/main/semconvgen) is (manually) created. However, this is something that should not be addressed in this PR 😉

Reply:
Definitely not this PR but TIL! Awesome!

Reply:
https://github.com/open-telemetry/opentelemetry-go-build-tools/pull/311/commits/285db0a72c2277fb6ccc2307ab0ee01f3dfc2049